shared/attributes.asciidoc

```diff
@@ -181,10 +181,13 @@ Elastic Cloud
 :ess:         Elasticsearch Service
 :ece:         Elastic Cloud Enterprise
 :eck:         Elastic Cloud on Kubernetes
-:serverless-full:  Elastic Cloud serverless
-:es-serverless: Elasticsearch on serverless
-:serverless-short: serverless
-:serverless-docs: https://docs.elastic.co/serverless
+:serverless-full:  Elastic Cloud Serverless
+:serverless-short: Serverless
+:es-serverless:    Elasticsearch Serverless
+:es3:              Elasticsearch Serverless
```

Comment:
@lcawl does it make sense to have both of these?

Reply:
I don't think it hurts since "es3" seems to be more common at the moment but "es-serverless" is more in alignment with the other projects (and more obvious IMO).